### PR TITLE
Box of assorted fixes & improvements

### PR DESCRIPTION
**Pull request by DaFuqs**
The fixes & improvements from https://github.com/DaFuqs/Spectrum/pull/539

### CODE REVIEW

File: src/main/resources/assets/spectrum/lang/en_gb.json [hunk 1]
```diff
@@ -1,3 +1,188 @@
 {
-  "advancements.spectrum.pluck_effulgent_feather.description": "§lOH LAWD OH SHUCKS OH SPIFFING OH BLIMEY OH SPLENDID"
+  "advancements.spectrum.collect_all_basic_pigments.description": "Collect all 12 kinds of colourful pigment",
+  "advancements.spectrum.collect_all_basic_pigments_besides_brown.description": "Mixing colours seemed so easy. But then brown had to ruin your fun. Literally impossible to mix using your current palette",
+  "advancements.spectrum.collect_gemstone_rock_candy.description": "Get a hold of Gemstone coloured Rock Candy",
+  "advancements.spectrum.collect_gemstone_rock_candy.title": "Colourful, Crunchy Confection",
+  "advancements.spectrum.collect_mysterious_locket.description": "A third colour combinations puzzle awaits you",
```
Comment (Noaaan) [inline src/main/resources/assets/spectrum/lang/en_gb.json]: Should this not be "colour combination puzzle"?

**Review by Noaaan — COMMENTED** [review]
Looks good. Found one possible improvement

File: src/main/resources/assets/spectrum/lang/en_gb.json (hunk 1 above)
Comment (Noaaan) [reply src/main/resources/assets/spectrum/lang/en_gb.json]:
```diff
-"advancements.spectrum.collect_mysterious_locket.description": "A third colour combinations puzzle awaits you",
+"advancements.spectrum.collect_mysterious_locket.description": "A third colour combination puzzle awaits you",
```
failed at getting the prompt from GitHub to allow you to change this in the web UI :pensive:

Comment (DaFuqs) [reply src/main/resources/assets/spectrum/lang/en_gb.json]: Yeah, agreed. I will change that ✅
Good catch!

**Pull request merged**